Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "stacker"
-version = "0.1.3"
+version = "0.2.0"
```

Comment:
I believe with recent changes this is no longer a breaking change (yay!) but I can also take care of this afte rlanding